docs(tutorial): add traffic light in planning simulation: (1) use AutowareStatePanel for engaging the vehicle (2) add traffic light simulation (#129)
autowarefoundation/autoware-documentation: docs/tutorials/ad-hoc-simulation/planning-simulation.md

## How to run a planning simulation

### Launch Autoware

```sh
source ~/autoware/install/setup.bash
ros2 launch autoware_launch planning_simulator.launch.xml map_path:=$HOME/Downloads/sample-map-planning vehicle_model:=sample_vehicle sensor_model:=sample_sensor_kit
```

Note that you cannot use `~` instead of `$HOME` here.

![after-autoware-launch](images/planning/lane-following/after-autoware-launch.png)

> ### Open Autoware State Panel

Reviewer: ```suggestion
### Add Autoware State Panel
```

> This panel is useful when we start planning simulation. Click `Panels -> Add new panel`, select `AutowareStatePanel`, and then press `OK`.

Reviewer: ```suggestion
This panel is useful when running planning simulations. To add the panel, click `Panels -> Add new panel`, select `AutowareStatePanel`, and then click `OK`.
```

![after-autoware-launch](images/planning/lane-following/open-autoware-state-panel.png)

### Set an initial pose for the ego vehicle

1. Click the `2D Pose estimate` button in the toolbar, or hit the `P` key.
2. In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the initial pose. The car should be placed in the right direction according to the lane(you can check the arrow on the lane). Now you will see a car on the Rviz.

![set-initial-pose](images/planning/lane-following/set-initial-pose.png)

### Set a goal pose for the ego vehicle

1. Click the `2D Nav Goal` button in the toolbar, or hit the `G` key

> 2. In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the goal pose. Keep in mind to set the goal pose aligned with the lane direction. Otherwise the path will not be planned. Now you will see a planned path.

Reviewer: Changed the warning about goal pose alignment to be a warning using the mkdocs admonition code block.

```suggestion
2. In the 3D View pane, click and hold the left-mouse button, and then drag to set the direction for the goal pose.  If done correctly, you will see a planned path from initial pose to goal pose.

!!! warning

    Remember to set the goal pose to be aligned with the lane direction, otherwise the path will not be planned! 
```

![set-goal-pose](images/planning/lane-following/set-goal-pose.png)

### Place a dummy object

You can skip this step.

To place a dummy car/pedestrian, click the `2D Dummy Car` and/or `2D Dummy Pedestrian` button in the toolbar, and then set its pose by dragging on the map. You can set its velocity in `Tool Properties -> 2D Dummy Car/Pedestrian` panel. In the image the velocity is set to 0.

![set-dummy-car](images/planning/lane-following/place-dummy-car.png)

> To delete those objects, click the `Delete All Objects` button in the toolbar.

Reviewer: ```suggestion
To delete any dummy objects placed in the view, click the `Delete All Objects` button in the toolbar.
```

### Set traffic light

You can skip this step.

To simulate traffic light recognition, go to `Panels -> Add new panel`, select `TrafficLightPublishPanel`, and then press `OK`. Then in the panel you need to set the `ID` and color of the traffic light.

You can check the `ID` of the traffic light by searching for `traffic` tag in `lanelet2_map.osm`. In the image there is one traffic light whose `ID` is 34836 and its color is `SET` to `RED`.

![set-traffic-light](images/planning/lane-following/set-traffic-light.png)

By clicking `PUBLISH` button the traffic light status is sent to the simulator, and the planned path changes accordingly.

![send-traffic-light-color](images/planning/lane-following/send-traffic-light-color.png)

### Engage the ego vehicle

> Now you can start driving by clicking `Engage` button in `AutowareStatePanel`. Or you can also manually `engage` the vehicle by running this command.

Reviewer: ```suggestion
Now you can start the ego vehicle driving by clicking the `Engage` button in `AutowareStatePanel`. Alteratively, you can manually engage the vehicle by running the following command:
```
